### ISSUE TITLE
Nocurses stuck

### ISSUE THREAD

**Issue by jonnieey** [andmarti1424/sc-im#697]
sc-im ran with  options --nocurses  option get stuck if  nocurses=0 is set in the configuration file.

**Comment by andmarti1424** [comment]
I believe you're overriding the `nocurses=0` when you set it with `--nocurses..`. It is also probably not stucked, but waiting for your commands. What are you trying to do btw?

**Comment by jonnieey** [comment]
Yeah, at first it go stuck but now its segmentation fault.  I'm tring to learn how to use sc-im without curses and also create more sc-im tutorials

**Comment by jonnieey** [comment]
![2022-03-23-160040](https://user-images.githubusercontent.com/47189744/159705216-b94da731-ecb1-4f65-a826-f886401e7ab4.png)

**Comment by andmarti1424** [comment]
could you share your scimrc file and the file your loading (if there is one)? thanks.

**Comment by jonnieey** [comment]
I'm not opening any file but here's my config file.

![2022-03-23-161013](https://user-images.githubusercontent.com/47189744/159707100-92a4600c-e9f8-44de-be7f-e86b5c22ecf8.png)
![2022-03-23-161031](https://user-images.githubusercontent.com/47189744/159707117-93084070-8ebe-4977-9a63-2dc6efefc6b1.png)

**Comment by andmarti1424** [comment]
Yes. Thats the problem. The `set nocurses=0` in the scimrc overrides (and collides) with the `--nocurses` passed as argv.
Your saying do not use ncurses interface and later you say "use it".

May be the solution would be to not allow setting that in the scimrc it if its already passed as argv..
Cause sc-im first get to know the argv parameters.. and then reads the config file.

**Comment by jonnieey** [comment]
Well, generally command line argument  overide configuration options. Yeah, I think command line args should take preceedence over config options.
should read config file first (the default), then if cmd args are provided overide those argurments in the config file by the cmd arg, else use the config file args.

**Comment by andmarti1424** [comment]
I commited a change to dev branch to now allow overloading nocurses that way. It should not crash anymore.
